Client/Frontend/UserContent/UserScripts/Playlist.js

```diff
@@ -273,10 +275,76 @@ window.__firefox__.includeOnce("$<Playlist>", function() {
                     return this.getAttribute('src')
                 },
                 set: function(value) {
+                    // Typically we'd call the original setter.
+                    // But since the property represents an attribute, this is okay.
                     this.setAttribute('src', value);
-                    $<notifyNode>(this);
+                    //$<notifyNode>(this); // Handled by `setAudioAttribute`
                 }
             });
+            
+            var setVideoAttribute = HTMLVideoElement.prototype.setAttribute;
+            HTMLVideoElement.prototype.setAttribute = function(key, value) {
+                setVideoAttribute.call(this, key, value);
+                if (key.toLowerCase() == 'src') {
+                    $<notifyNode>(this);
+                }
+            }
+            
+            var setAudioAttribute = HTMLAudioElement.prototype.setAttribute;
+            HTMLAudioElement.prototype.setAttribute = function(key, value) {
+                setAudioAttribute.call(this, key, value);
+                if (key.toLowerCase() == 'src') {
+                    $<notifyNode>(this);
+                }
+            }
+            
+            // FOR DEBUGGING ONLY!
+            
+            /*var oldAudio = Audio;
```

Comment:
Do we want to expose it to users? Afaik we do not run this script through webpack to remove such comments